.github/ga_packages/build_cache_executor/src/utils/common.ts

```diff
@@ -0,0 +1,71 @@
+import * as folderHash from "folder-hash";
+import * as crypto from "crypto";
+import * as fs from "fs-extra";
+
+export interface outputStream {
```

Comment:
That's a weird name for what appears to be essentially a logger - should we name it `logger`?